### ISSUE TITLE
Chrome Extension not logging in/loading 

### ISSUE THREAD

**Issue by gabibeann** [bitwarden/clients#11133]
### Steps To Reproduce

When logging into Chrome extension with my master password, the extension closes and kicks me to the e-mail entry screen. It will not complete my login or take me to my vault. The extension icon (bitwarden logo) is also gray instead of blue?

### Expected Result

My master password will log me in and I can use the extension to auto-fill on other websites etc.

### Actual Result

The extension will not complete the log-in and load my vault after I enter my master password. I am still able to log into the web app.

### Additional Context

I have browsed the community forum for similar issues; I did remove the extension, close/re-open chrome, add the extension again, and this did not help. I also made sure to check my login credentials, even copying them from a different program to make sure they were accurate and I was not making any errors (also mentioned in other forum posts). There are two reviews on the Chrome webstore mentioning similar issue from Sept 18, 2024 as well. Ping tests fine

### Operating System

Windows

### Operating System Version

Windows 10 Enterprise 22H2

### Web Browser

Chrome

### Browser Version

128.0.6613.138 (Official Build) (64-bit)

### Build Version

2024.9.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by jtodddd** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by gabibeann** [comment]
Hi,

I had this issue yesterday (9/18) & was not able to use the extension at
all, but today I have been able to log in on the chrome extension.

**Comment by BlackMercy** [comment]
I'm seeing weird issue on Edge browser extension as well. I can unlock the vault using master password, but if I click "Unlock with biometrics", nothing will happen. No Windows Hello popup. This started today. Unlocking Bitwarden Desktop using biometrics works normally.

I'm using extension v.2024.9.1 and Bitwarden desktop v.2024.8.2. Edge v.128.0.2739.67. I saw on the Releases page that desktop client v.2024.9.0 was released yesterday but mine says I'm on the lastest version.

**Comment by oliverjnash** [comment]
> I'm seeing weird issue on Edge browser extension as well. I can unlock the vault using master password, but if I click "Unlock with biometrics", nothing happens. No Windows Hello popup. This started today. Unlocking Bitwarden Desktop using biometrics works normally.
> 
> I'm using extension v.2024.9.1 and Bitwarden desktop v.2024.8.2. Edge v.128.0.2739.67. I saw on the Releases page that desktop client v.2024.9.0 was released yesterday but mine says I'm on the lastest version.

Just to add, I'm in exactly the same boat with the same software versions. Biometrics on the Edge extension haven't been working for 24 hours or so. Have just updated Edge to 129.0.2792.52 and it's still the same. Both the extension and the desktop client are saying they're up to date. Have tried uninstalling and re-installing the extension to no avail.

**Comment by bobvandevijver** [comment]
It looks like the following line just times out:

https://github.com/bitwarden/clients/blob/992964fc08e170f4d466d7dac342f5ac622bc909/apps/browser/src/platform/services/foreground-browser-biometrics.ts#L18

Which causing an exception because the `response` is undefined.

![image](https://github.com/user-attachments/assets/d5d74c0f-ff3b-4e30-b7b7-b719a19acb9d)

Edge version 128.0.2739.79, the rest is equal.

**Comment by efa2d19** [comment]
> I'm seeing weird issue on Edge browser extension as well. I can unlock the vault using master password, but if I click "Unlock with biometrics", nothing happens. No Windows Hello popup. This started today. Unlocking Bitwarden Desktop using biometrics works normally.

Not Windows and Edge problem only, seeing the same stuff on MacOS and Arc

**Comment by plexecutor** [comment]
> I'm seeing weird issue on Edge browser extension as well. I can unlock the vault using master password, but if I click "Unlock with biometrics", nothing happens. No Windows Hello popup. This started today. Unlocking Bitwarden Desktop using biometrics works normally.
> 
> I'm using extension v.2024.9.1 and Bitwarden desktop v.2024.8.2. Edge v.128.0.2739.67. I saw on the Releases page that desktop client v.2024.9.0 was released yesterday but mine says I'm on the lastest version.

I am experiencing this issue as well. Bitwarden Desktop+extension 2024.9.1, and both edge v128 and the new v129.

**Comment by ImagineReason** [comment]
I've been having the same problem for several days, only it's Firefox on Windows 10.

**Comment by plexecutor** [comment]
> I've been having the same problem for several days, only it's Firefox on Windows 10.

Yeah this has been brutal with a 5 word hyphenated pass phrase. After about the 10th time of typing it I got frustrated and disabled lock entirely. I hope this is fixed soon.

**Comment by BlackMercy** [comment]
> I'm seeing weird issue on Edge browser extension as well. I can unlock the vault using master password, but if I click "Unlock with biometrics", nothing happens. No Windows Hello popup. This started today. Unlocking Bitwarden Desktop using biometrics works normally.
> 
> I'm using extension v.2024.9.1 and Bitwarden desktop v.2024.8.2. Edge v.128.0.2739.67. I saw on the Releases page that desktop client v.2024.9.0 was released yesterday but mine says I'm on the lastest version.

For anyone facing this issue, this [comment in another thread](https://github.com/bitwarden/clients/issues/11168#issuecomment-2365239211) says that manually updating the desktop client to v.2024.9.0 solves the problem. I can confirm that I can unlock the Edge extension vault with Windows Hello now.

**Comment by bobvandevijver** [comment]
Manually updating the desktop client to 2024.9.0 has indeed solved the issue here! 👍🏻

**Comment by ImagineReason** [comment]
I was on the latest version already.  I had the same problem two versions ago.  I solved it by disabling Autofill in Firefox.

The problem actually sprang up in Edge as well.

**Comment by dyxushuai** [comment]
> updating the desktop client to 2024.9.0 has indeed solved the iss

But "Allow browser integration" in macOS needs it comes from appstore, the version in appstore is still 2024.8.2

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
